.idea/$PRODUCT_WORKSPACE_FILE$

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As a rule you probably don't want to commit a bunch of IDE-specific files unless there's good reason for it.